.gitignore

```
@@ -1,3 +1,8 @@
+# built cheatsheets and handouts
+# ----------------------------------
+cheatsheets*.p[dn][fg]
+handout-*.p[dn][fg]
```

This also hits ,pdg and .pnf, which probably don't exist, but better to be explicit
```suggestion
cheatsheets*.pdf
cheatsheets*.png
handout-*.pdf
handout-*.png
```

---

Agreed, better to be explicit. This has been fixed.